config/base/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-ingress
```

Comment:
Do we want to also force users into using the `nginx-ingress` namespace?

Reply:
it's what we do already